core/src/main/scala/kafka/log/LogCleanerManager.scala

```diff
@@ -64,6 +64,19 @@ private[log] class LogCleanerManager(val logDirs: Array[File], val logs: Pool[To
   @volatile private var dirtiestLogCleanableRatio = 0.0
   newGauge("max-dirty-percent", new Gauge[Int] { def value = (100 * dirtiestLogCleanableRatio).toInt })
 
+  /* a gauge for tracking the number of logCleaner runs */
+  @volatile private var numLogCleanRuns = 0
+  newGauge("num-runs", new Gauge[Int] { def value = numLogCleanRuns })
+
+  /* a gauge for tracking the last time of logCleaner run */
+  @volatile private var logCleanLastRun : Long = System.currentTimeMillis
+  newGauge("last-run-time", new Gauge[Long] { def value = logCleanLastRun })
+
+  /* a gauge for tracking the number of filthy logs. A non zero value indicates
```

Comment:
How about "A non zero value for an extended period of time indicates that the cleaner has not been successful in cleaning the logs."